scripts/validate_docstrings.py

```diff
@@ -331,6 +336,33 @@ def parameter_mismatches(self):
 
         return errs
 
+    @contextlib.contextmanager
+    def _write_examples_code_to_temp_file(self):
+        """
+        Generate file with source code from examples section.
+        """
+        content = ''.join(('import numpy as np; '
+                           'import pandas as pd  # noqa: F401,E702\n',
+                           *self.examples_source_code))
+        with tempfile.NamedTemporaryFile(mode='w') as file:
+            file.write(content)
+            file.flush()
+            yield file
+
+    def validate_pep8(self):
+        if self.examples:
+            with self._write_examples_code_to_temp_file() as file:
+                application = flake8.main.application.Application()
+                application.initialize(["--quiet"])
+                application.run_checks([file.name])
+                application.report()
+
+            for statistic in application.guide.stats.statistics_for(''):
+                if statistic.message.endswith('from line 1'):
+                    statistic.message = "It's assumed that pandas and numpy" \
+                                        " are imported as pd or np"
+                yield statistic
```

Comment:
I guess with the original `yield from` we may get a pep8 error of duplicate import, or something similar. I'm fine with that. Besides the pep8 errors, we'll have the error of the other section implemented in #23161, which should make it very easy for users to identify the problem.

So, I'd simply leave the `yield from ...`

Reply:
I found `F811 redefinition of unused 'np' from line 1` could be confusing when read first.